recipe/build.sh

```diff
@@ -1,5 +1,40 @@
 #!/bin/bash
-export DISABLE_AUTOBREW=1
```

Comment:
We might as well simplify the script since we're editing. The `if...else` is from older `conda_r_skeleton_helper` versions. What you had was fine, just remove the `installspec`.